packages/ui-kit-theme/src/index.ts

```diff
@@ -16,13 +17,15 @@ export { palette, spacing };
 
 export const light = createTheme({
   components: LIGHT_COMPONENTS,
+  ...breakpoints,
```

Comment:
From the above comment: It could be added to the theme object as `breakpoints,` instead of `...breakpoints`